### ISSUE TITLE
[feature request] Retry

### ISSUE THREAD

**Issue by coolinger** [sabaatworld/HyperionScreenCap#53]
Hi, after wakeup from suspend, and sometimes during the day, the connection is broken. A simple retry setting would be really nice!

Thanks for the great work!

**Comment by sabaatworld** [comment]
Agreed. I'm pretty busy these days with work but I'll remember to add this when I get time.

**Comment by SunnyMoo** [comment]
Hi, I second that. An option to be able to stop/start on double click instead of stopping or starting would be nice. Thanks !

**Comment by sabaatworld** [comment]
Newly released v2.9 has modified retry behavior which might fix this issue. There are some retries built into the app already.
